vocab/properties.json

```diff
@@ -735,7 +729,7 @@
         ]
     },
     "hasAlternativeVersion": {
-        "description": null,
+        "description": "Reference to a particular form of something differing only in the format, but not the actual content.",
```

Comment:
I don't think this description is correct. Generally it is the content that differs between versions. Sometimes the format differs, but not usually.

Reply:
This will be our specific definition for "alternative" versions, but I agree that "format" not necessarily covers it. For me an alternative version provides essentially the same data but presented in a different form or context. E.g. one dataset was delivered in a custom structure, and then they present it again BIDS conform. Or one delineation set was presented in reference space A, and a second time in reference space B. 

Do you have an idea how to describe this best?

Reply:
Suggestion: "Reference to an alternative form of something where the essence of the original form remains unchanged. "